use-the-built-in-sql-client.md

```diff
@@ -18,6 +18,7 @@ $ ./cockroach sql <flags>
 # Execute SQL from the command line:
 $ ./cockroach sql --execute="<sql statement>;<sql statement>" --execute="<sql-statement>" <flags>
 $ echo "<sql statement>;<sql statement>" | ./cockroach sql <flags>
+$ ./cockroach sql <flags> < file-containing-statements.sql
```

Comment:
using `<flags>` could be confusing for some since there's also a legitimate `<` in the command. Maybe `[flags]`? Probably clear enough as it is.